server/embedded/README.md

```diff
@@ -0,0 +1,42 @@
+# embedded
+LabKey Server can now be started with embedded Tomcat. This feature is in development phase and not ready for use in production. Free versions of IntelliJ may lack support for Spring boot debug configurations required to run the embedded Tomcat.
```

Comment:
Can we make the warnings bolded?  People tend to miss these things and it gives rise to more questions.